scss/grid/_flex-grid.scss

```diff
@@ -119,6 +109,11 @@
 
   flex: flex-grid-column($columns);
 
+  // fixes recent Chrome version not limiting child width
+  // https://stackoverflow.com/questions/34934586/white-space-nowrap-and-flexbox-did-not-work-in-chrome
+  @if $columns == expand {
+    min-width: 0;
+  }
```

Comment:
Even if I understand the logic behind this move (all fixes in the same place), this Chrome fix should stay in the component mixin `flex-grid-column`, because it does not need to be re-generated when we apply a different size.

For example:
```sass
.my-flex-col {
    @include flex-grid-column(12);
    // ✓ The IE fix is generated for 12 columns.
    // ✓ The Chrome fix is generated.
}

.my-flex-col--demi {
    @include flex-grid-size(6);
    // ✓ The IE fix must be generated for 6 columns.
    // ☓ The Chrome fix still work, so is useless here.
}
```

Please move back the Chrome fix to `flex-grid-column`.